spec/controllers/v2/content_items_controller_spec.rb

```diff
@@ -5,10 +5,37 @@
 
   before do
     stub_request(:any, /content-store/)
-    @draft = FactoryGirl.create(:draft_content_item, content_id: content_id)
+    @draft = FactoryGirl.create(:draft_content_item, content_id: content_id, format: "topic")
     FactoryGirl.create(:version, target: @draft, number: 2)
   end
 
+  describe "index" do
+    context "with pagination params" do
+      before do
+        get :index, content_format: 'topic', fields: ['content_id'], start: "0", count: "20"
+      end
+      it "is successful" do
+        expect(response.status).to eq(200)
+      end
+      it "responds with the content item as json" do
+        parsed_response_body = JSON.parse(response.body)
+        expect(parsed_response_body.first.fetch("content_id")).to eq("#{content_id}")
```

Comment:
No need to use string substitution here (or below), `content_id` is already a string.